pkgs/ko1nksm/getoptions/gengetoptions/registry.yaml

```diff
@@ -0,0 +1,12 @@
+packages:
+  - type: github_release
+    name: ko1nksm/getoptions/gengetoptions
+    repo_owner: ko1nksm
+    repo_name: getoptions
+    asset: gengetoptions
+    description: An elegant option/argument parser for shell scripts (full support for bash and all POSIX shells)
+    supported_envs:
+      - linux
+      - darwin
```

Comment:
Windows also seems to be supported.

https://github.com/ko1nksm/getoptions#features

> High portability, supports all platforms (Linux, macOS, Windows, etc) where works POSIX shells

Reply:
But other packages made by shell script in aqua-registry exclude windows.

Reply:
I see.
Hmm. Actually, there is no strict policy about these case.
Okay. Let's exclude Windows for now.
I'll check if this works on Windows when I'm free.

BTW, there is no way to list up packages made by shell scripts to check them...